src/scala/com/github/johnynek/bazel_deps/maven/Tool.scala

```diff
@@ -81,16 +83,22 @@ object Tool {
           props.getOrElse(k, "${" + k + "}")
         })
       }
+      def ropt(s: Option[String]): Option[String] = s match {
```

Comment:
isn't this just `s.map(r)`? can we use that instead?

Reply:
Doh. Yes, thanks.